builtin/logical/mysql/backend_test.go

```diff
@@ -156,8 +184,8 @@ func TestBackend_roleCrud(t *testing.T) {
 		Backend: b,
 		Steps: []logicaltest.TestStep{
 			testAccStepConfig(t, connData, false),
-			testAccStepRole(t),
-			testAccStepReadRole(t, "web", testRole),
+			testAccStepRole(t, false),
+			testAccStepReadRole(t, "web", testRoleHost),
```

Comment:
Wouldn't the equivalent here of the previous test be to use `testRoleWildcard`? It seems like you're changing to test the new value but now leaving the old value untested.